data/helpers/dbHelper.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../dbConfig.js");
+
+module.exports = {
+  findUserById,
+  findUserByName,
+  registerUser,
+  getUsers,
+  getUsersByDept
+};
+
+function findUserById(id) {
+  return db("users")
+    .where({ id })
+    .first();
+}
+
+function findUserByName(username) {
+  return db("users")
+    .where({ username })
+    .first();
+}
+
+async function registerUser(user) {
+  const newUser = await db("users").insert(user);
+  return findUserById(newUser[0]);
```

Comment:
You could alternatively pull the first element off of the array with [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring):
```javascript
  const [ newUser ] = await db("users").insert(user);
  return findUserById(newUser);
```
I would also consider renaming the variable to reflect the fact that its an id (and not a user object):
```javascript
  const [ newUserId ] = await db("users").insert(user);
  return findUserById(newUserId);
```